### PR TITLE
Addition in  navbar

### PR DESCRIPTION
**Pull request by DivyanshiNarang**
Resolves issue: #25

## Description of Changes
Added Home, About, and Contact Us in Navbar.
@SrijanShovit  Please provide me with the respective pages for the linking.

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

##Screenshots

https://github.com/SrijanShovit/carbonops-v2/assets/74779601/d9183f66-0133-4518-8d64-e494dd3301bb

In mobile view:

https://github.com/SrijanShovit/carbonops-v2/assets/74779601/4080aaee-f272-484c-b511-b7f235779183

### CODE REVIEW

**Review by SrijanShovit — COMMENTED** [review]
Hello DivyanshiNarang, thanks for rising a Pull request, your contribution is valuable to us. The maintainers will review this Pull Request and provide feedback as soon as possible. Keep the great work up!<p></p>

File: components/Navbar.js [hunk 1]
```diff
@@ -45,6 +45,24 @@ const Navbar = () => {
             </a>
             {/* Desktop Navmenu */}
             <nav className="hidden md:flex justify-center items-center gap-8">
+              <Link
+                href="#"
```
Comment (shreya024) [inline components/Navbar.js:49]: onClick route to /

File: components/Navbar.js [hunk 2]
```diff
@@ -45,6 +45,24 @@ const Navbar = () => {
...
+                className="text-lg mx-3 hover:text-[#00BD57] ease-out duration-200"
+              >
+                HOME
+              </Link>
+              <Link
+                href="#"
```
Comment (shreya024) [inline components/Navbar.js:55]: onClick route to /aboutus

File: components/Navbar.js [hunk 3]
```diff
@@ -45,6 +45,24 @@ const Navbar = () => {
...
+                className="text-lg mx-3 hover:text-[#00BD57] ease-out duration-200"
+              >
+                ABOUT
+              </Link>
+              <Link
+                href="#"
```
Comment (shreya024) [inline components/Navbar.js:61]: onClick redirect to /contactus

**Comment by DivyanshiNarang** [comment]
Hey @shreya024, I have done the required changes. Please review it.

**Review by shreya024 — APPROVED** [review]

**Pull request merged**